server/api/db/sync.js

```diff
@@ -0,0 +1,16 @@
+import { sequelize } from "@/server/libs/sequelize";
+import { Todo } from "@/server/models/todo";
+import { User } from "@/server/models/user";
+
+// eslint-disable-next-line no-undef
+export default defineEventHandler(async () => {
+    User.hasMany(Todo);
+    Todo.belongsTo(User);
```

Comment:
À mettre directement dans tes modèles.

Documentation : https://sequelize.org/docs/v6/advanced-association-concepts/creating-with-associations/

Reply:
Si je comprends bien la doc, la relation One to many n'est pas créée dans le modèle mais au moment de créer (`.create`) une instance ?

Reply:
J'ai suivi cette doc : [Sequelize- assoc](https://sequelize.org/docs/v6/core-concepts/assocs/)

Reply:
Oui mais faut que tu mettes ce morceau de code dans le modèle car c'est justement lié à l'instance du modèle que tu exportes